js/main.js

```diff
@@ -1,141 +0,0 @@
-// API key set globally so that it is easily edited if needed
-const apiKey = '7e8e7a9139e4f09fb253db0d8ddce620';
```

Comment:
when we have constants like these it's good practice `API_KEY` at the top of the file.